### PR TITLE
Remove hint

### PR DESCRIPTION
**Pull request by npm1**
***
<a href="https://pr-preview.s3.amazonaws.com/npm1/FedCM/pull/285.html" title="Last updated on Jun 23, 2022, 2:08 PM UTC (ca60ca6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/285/cc956a4...npm1:ca60ca6.html" title="Last updated on Jun 23, 2022, 2:08 PM UTC (ca60ca6)">Diff</a>

### CODE REVIEW

**Review by cbiesinger — APPROVED** [review]

File: spec/index.bs [hunk 1]
```diff
@@ -358,7 +358,6 @@ could be implemented.
         providers: [{
           url: "https://idp.example/manifest.json",
           clientId: "123",
-          hint: id,
```
[the remark this answers is not in the document]
Comment (npm1) [reply spec/index.bs:361]: I imagine you mean the multi-login example. Done

**Comment by samuelgoto** [comment]
humm apparently, there is a conflict. can you sync and i'll merge then?

**Comment by npm1** [comment]
> humm apparently, there is a conflict. can you sync and i'll merge then?

Done

**Comment by npm1** [comment]
> there is still a conflict?

From the PR you had just merged?
Done.

**Pull request merged**